### ISSUE TITLE
Installer Icon

### ISSUE THREAD

**Issue by cor3ntin** [aseba-community/aseba#751]
Currently the windows installer uses the default NSIS icon. Should we provide an icon for the overall aseba installer ?

**Comment by stephanemagnenat** [comment]
Yes we should. We currently lack an Aseba icons. I had some prototypes but they did not receive popular enthusiasm so far. I created a new PR (#772) to track that question.

**Comment by stephanemagnenat** [comment]
I agree, I changed the title accordingly.
